package.json

```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "@croct/json": "^1.1.0",
-    "@croct/sdk": "^0.9.1",
+    "@croct/sdk": "^0.10.0",
```

Comment:
Shouldn't we leave this bump to dependabot. There isn't sdk 0.10.0 yet.

Reply:
Fixed :)

Reply:
Nevermind, new version just released